packages/protocol/contracts/common/MetaTransactionWallet.sol

```diff
@@ -56,7 +67,7 @@ contract MetaTransactionWallet is
    * @return The storage, major, minor, and patch version of the contract.
    */
   function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
-    return (1, 1, 0, 1);
+    return (1, 1, 1, 1);
```

Comment:
If the minor number increases, shouldn't patch be reset to 0? Why didn't CI catch this?

Reply:
Ya, good point.

Reply:
Looks like MetaTransactionWallet.sol is [excluded](https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/scripts/bash/check-versions.sh#L48) from the version checks.

@yorhodes @m-chrzan  - Was this intentional?